AdyenActions/Actions/VoucherAction.swift

```diff
@@ -81,12 +89,25 @@ public enum VoucherAction: Decodable {
             return action
         case let .econtextStores(action):
             return action
+        case let .oxxo(action):
+            return action
         }
     }
 }
 
+/// :nodoc:
+/// Describes a voucher that has an instructions url.
+internal protocol InstructionAwareVoucherAction {
+    
+    /// :nodoc:
+    /// The instruction url.
+    var instructionsUrl: String { get }
```

Comment:
would be nice if this was of type `URL` and not `String`
maybe in the decoding of `GenericVoucherAction` we could check if a `URL` can be constructed from `instructionsUrl`. this way in `VoucherComponentExtensions` we can make `createReadInstructionsAlertAction` return a non-optional `UIAlertAction`.

Reply:
👍